src/main/kotlin/org/rust/ide/presentation/Ty.kt

```diff
@@ -67,6 +67,7 @@ private fun render(
             is TyInfer.FloatVar -> float
         }
         is FreshTyInfer -> "<fresh>" // really should never be displayed; debug only
+        is TyAnon -> ty.traits.map { it.element.name ?: anonymous }.joinToString()
```

Comment:
Hm, it's interesting question - how we should display `TyAnon`. I think it should be displayed as close to it's syntax as possibe, i.e. `impl A+B+C`. So I think we should write here something like `ty.traits.map { it.element.name ?: anonymous }.joinToString("+", "impl ")`

Reply:
And this code isn't tested. Let's add one simple test to `RsTypeResolvingTest`! :)

Reply:
Sounds good. Done.